.github/workflows/test.yml

```
@@ -0,0 +1,92 @@
+# Terraform Provider testing workflow.
+name: "Tests"
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
```

We should also have this run on merges to `main`. Also, we probably need to have this run even when the readme changes because we are going to use it as a required status check for merging PRs.
```suggestion
on:
  pull_request:
  push:
    branches: [main]
```